.eslintrc.json

```diff
@@ -59,6 +59,8 @@
 		"comma-dangle": "error",
 		"keyword-spacing": "error",
 		"space-in-parens": ["error", "never"],
-		"func-style" : ["warn", "expression"]
+		"func-style" : ["warn", "expression"],
+		"require-await": "off",
+		"@typescript-eslint/no-floating-promises": ["error"]
```

Comment:
This will error out when we are missing an `await` for a returned promise function